Description: Given that the only 'localhost' entry in default Debian
 /etc/hosts resolves to 127.0.0.1, this enables http://localhost:631
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2014-02-23
--- a/data/cups.socket.in
+++ b/data/cups.socket.in
@@ -3,6 +3,8 @@
 
 [Socket]
 ListenStream=@CUPS_DEFAULT_DOMAINSOCKET@
+# This line is mandatory in Debian where localhost only resolves to ipv4's 127.0.0.1
+ListenStream=127.0.0.1:631
 ListenStream=631
 BindIPv6Only=ipv6-only
 
